packages/p2p/src/components/order-details/order-details.scss

```diff
@@ -56,6 +57,11 @@ $card-width: 456px;
         padding: 1rem 2.4rem;
     }
 
+    &--padding:last-of-type {
+        margin-bottom: 0px !important;
```

Comment:
Is there any way we can avoid `!important` here?

Reply:
Yeap, I'll refactor it using `unset`